app/src/main/res/menu/profile.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
-    <group android:id="@+id/profileMenu">
+    <group android:id="@+id/profile_menu">
         <item
             android:id="@+id/orgaApp"
```

Comment:
here

Reply:
It was only for group ids I presume

Reply:
@srv-twry @iamareebjamal What should be done

Reply:
https://stackoverflow.com/questions/35923922/best-practices-for-id-naming-conventions-in-android

Every id should be in snakecase.

Reply:
But then we have always used camel case for previous ids!

Reply:
@iamareebjamal will have the final say. I am still in favour of snake case for everything in xml.

Reply:
For the ones you are referring in kotlinx should remain in camel case, i.e., view IDs. Others should be in snake case

Reply:
I was informed on another PR that "view ids should remain to be camel case as they are picked up by kotlinx".